docs/index.rst

```diff
@@ -51,8 +51,15 @@ The ME-ICA pipeline originally performed both pre-processing and TE-dependent
 analysis of multi-echo fMRI data; however, ``tedana`` now assumes that you're
 working with data which has been previously preprocessed.
 
-.. _ME-ICA: https://github.com/me-ica/me-ica
+For a summary of multi-echo fMRI, which is the technology ``tedana`` builds on, 
+visit `Multi-echo fMRI`_. 
+
+For a detailed procedure of how ``tedana`` analyzes the data from multi-echo fMRI, 
+visit `Processing pipline details`_.
 
+.. _ME-ICA: https://github.com/me-ica/me-ica
+.. _Multi-echo fMRI: https://tedana.readthedocs.io/en/latest/multi-echo.html
+.. _Processing pipline details: https://tedana.readthedocs.io/en/latest/multi-echo.html
```

Comment:
This points to the multi-echo page, not the approach page.

Reply:
Apologies!